build.gradle

```diff
@@ -533,7 +533,11 @@ def sonarExclusions = [
   '**uk/gov/hmcts/reform/prl/config/**/*',
   '**uk/gov/hmcts/reform/prl/utils/SslVerificationDisabler.*',
   '**uk/gov/hmcts/reform/prl/Application.*',
-  '**uk/gov/hmcts/reform/prl/utils/ElementUtils.java'
+  '**uk/gov/hmcts/reform/prl/utils/ElementUtils.java',
+  '**src/main/java/uk/gov/hmcts/reform/prl/enums/manageorders/*',
+  '**src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java',
+  '**src/main/java/uk/gov/hmcts/reform/prl/controllers/ManageOrdersController.java',
+
```

Comment:
Why they are added to the exclusion list?

Reply:
Added coverage and removed form the list